data/data/rhcos-4.8.json

```diff
@@ -0,0 +1,394 @@
+{
+  "stream": "rhcos-4.8",
```

Comment:
What is the significance of the name of the stream? Does the file name have to match the name of the stream, or is that just convention?

Reply:
Convention, see e.g.
https://builds.coreos.fedoraproject.org/streams/stable.json
as another example.

Here we're matching the OpenShift versioning.

Reply:
My concern is the maintenance involved with code and docs if this file changes names with every release.

Even with this PR where the name change is fresh, there are a number of locations in this repo where the old rhcos.json name is still used.
```
$ grep -rnE . -e "rhcos\.json" --exclude-dir vendor --exclude-dir .git
./pkg/types/vsphere/platform.go:27:	// ClusterOSImage overrides the url provided in rhcos.json to download the RHCOS OVA
./images/installer/Dockerfile.upi.ci:17:COPY --from=builder /go/src/github.com/openshift/installer/data/data/rhcos-4.8.json /var/lib/openshift-install/rhcos.json
./data/data/install.openshift.io_installconfigs.yaml:1760:                    description: ClusterOSImage overrides the url provided in rhcos.json
./docs/user/azure/install_upi.md:201:export VHD_URL=`curl -s https://raw.githubusercontent.com/openshift/installer/release-4.3/data/data/rhcos.json | jq -r .azure.url`
./docs/user/azure/install_upi.md:207:export VHD_URL=`curl -s https://raw.githubusercontent.com/openshift/installer/master/data/data/rhcos.json | jq -r .azure.url`
./docs/user/metal/customization_ipi.md:28:[/data/data/rhcos.json](/data/data/rhcos.json), but they can be
./docs/user/metal/customization_ipi.md:32:uncompressed SHA256 from rhcos.json.
./docs/user/gcp/install_upi.md:473:export IMAGE_SOURCE=$(curl https://raw.githubusercontent.com/openshift/installer/master/data/data/rhcos.json | jq -r .gcp.url)
./docs/user/openstack/customization.md:115:Normally the installer downloads the RHCOS image from a predetermined location described in [data/data/rhcos.json](/data/data/rhcos.json)). But the download URL can be overridden, notably for disconnected installations.
```